code/game/objects/items/oddities.dm

```diff
@@ -645,7 +649,7 @@
 /obj/item/oddity/code_book
 	name = "Marshal Code Handbook"
 	desc = "A heavy book with details and translations of the Marshal codes."
-	icon_state = "book_code"
+	icon_state = "instructional_rob" // Fits better with a supposed Marshal's logo up front. - Seb
```

Comment:
I dislike changing it to that sprite, as the code-book one has refences to each paragraph within its sprite

Reply:
![image](https://user-images.githubusercontent.com/1294508/188973769-12344d50-4013-4ee3-9a30-0daa707d6de1.png)

Out of these two, the one that has the sheriff badge makes the most sense to me as a codespeak manual, that's why